apps/full-site-editing/full-site-editing-plugin/dotcom-fse/editor/style.scss

```diff
@@ -35,6 +35,45 @@
 		}
 	}
 
+	// Taken from styles/starter-page-templates-editor.scss
+	$breakpoint-mobile: 660px;
+	$breakpoint-tablet: 783px;
+
+	.edit-post-header__toolbar {
+		// The right side of the header has 16px,
+		// the left side should too, otherwise it looks out of balance.
+		// Applies to all screen sizes.
+		padding-left: 16px;
+	}
+
+	@media screen and ( max-width: $breakpoint-tablet ) {
+		// Removes whitespace above header in mobile & tablet view.
+		.block-editor-editor-skeleton {
```

Comment:
Perhaps this is the culprit for why we get hidden under the wp-admin bar?

Reply:
Thanks! Fixed in this commit: https://github.com/Automattic/wp-calypso/pull/40361/commits/a7295c3a1c3dd248295d70cd1e18e6a765bcdfd0

Reasoning for the top whitespace can be traced back to: https://github.com/WordPress/gutenberg/pull/13425